public/app/features/alerting/unified/utils/rule-form.ts

```diff
@@ -71,7 +73,7 @@ export const getDefaultFormValues = (): RuleFormValues => {
     uid: '',
     labels: [{ key: '', value: '' }],
     annotations: defaultAnnotations,
-    dataSourceName: null,
+    dataSourceName: GRAFANA_RULES_SOURCE_NAME, // let's use Grafana-managed alert rule by default
```

Comment:
I noticed that – for some reason – this is `null` when creating or editing Grafana-managed rules but non-empty when doing the same for data source managed rules. Just a weird inconsistency this fixes, hopefully without adverse effects I'm not aware of.